docs/drill4net/support-matrix.mdx

```diff
@@ -0,0 +1,54 @@
+﻿---
+id: support-matrix
+title: Support Matrix
+---
+import { Image } from '@components';
+import { Info } from '@components';
+import { Table } from '../../src/components'
+
+## Testing Frameworks
+
+<Table>
+
+| Framework |      Supported versions     |
+|:---------:|:---------------------------:|
+| NUnit     |	    - 3.13.3			  |
+| XUnit     | 2.4.2 - 4.13.2              |
+| Selenium  |	    - 4.12.4			  |
+| Playwright|		- 1.32.0			  |
+
+</Table>
+
+## .NET Frameworks
+
+<Table>
+
+| Framework |      Supported versions     |
+|:---------:|:---------------------------:|
+| .NET			 | 5, 6, 7			   	  |
+| .NET Core      | 1.0 - 3.1              |
+| .NET Framework | 4.7.2, 4.8, 4.8.1	  |
+
+</Table>
+
+<Info>
+  NOTE <br/>
+  Currently, Selenium with .NET 7 is not supported 
+</Info>
+
+## Operating systems
+
+- Linux  
```

Comment:
Linux has lots of distributions / versions - please specify exact distribution and version the solution were tested on (e.g. "Ubuntu 20.0.1", "Debian X.Y.Z" etc)
It is not required to perform exhaustive testing of _all_ known distributions - just make sure to write down those version we've tested .NET agent (scanner/injector) on